src/components/common/FileUploadPdf.vue

```diff
@@ -0,0 +1,295 @@
+<template>
```

Comment:
This component was originally FileUploadPreview.vue from Create UI, but I renamed it and refactored it heavily to support single, repeatable, form-like validation.